quickstarts/nextcloud/config.yml

```diff
@@ -0,0 +1,39 @@
+slug: nextcloud
+
+description: |
+  ## Comprehensive monitoring quickstart for Nextcloud
+  With New Relic's Nextcloud monitoring system, keep an eye on your Nextcloud in real-time, correlate transactions moving across the application environment and troubleshoot the issue which you are facing.
+  ### Why monitor Nextcloud?
+  Control, monitor and secure all the data of Nextcloud with Infrastructure monitoring agent and Prometheus open metric integration to view a thorough Nextcloud server dashboard that contains a high-level overview of their app's performance. 
+  - Assist teams in creating architectural maps, tracking down all the issues fast, and observing the Apdex of the app.
+  - Maintain a quick throughput and low latency.
```

Comment:
"Use our tools to help you maintain quick throughput and low latency on your server."